src/runtime/signal.c

```diff
@@ -0,0 +1,89 @@
+//go:build none
+
+// Ignore the //go:build above. This file is manually included on Linux and
+// MacOS to provide os/signal support.
```

Comment:
Can this just have `//go:build linux || darwin`?

Reply:
Unfortunately not. Using it as a regular C file would imply CGo, which forces an import of the syscall package (not needed in TinyGo but that's how the Go toolchain works), which then imports the runtime package. So doing it this way avoids a circular dependency.

The same is true for runtime_unix.c (which is included in the same way).